src/components/input/date/index.tsx

```diff
@@ -38,7 +38,9 @@ export class InputDate implements ComponentWillLoad, Clearable, Input {
 	@Event() smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color) => void>
 
 	componentWillLoad(): void {
-		this.smoothlyInputLoad.emit(() => {})
+		this.smoothlyInputLoad.emit(() => {
+			return
```

Comment:
the return is unnecessary and it adds two extra lines. Can we keep it as `() => {}`

Reply:
no, it errors as "unexpected empty arrow function. I do agree that it we should come up with a nicer way to emit ourselves.

Reply:
Apparently you can just emit empty!